### PR TITLE
Flash messages

### PR DESCRIPTION
**Pull request by stevegraham**
- Add flash messages for info, notice, and error
 - Add logout message

### CODE REVIEW

**Comment by stevegraham** [comment]
@michaelelwin need copy buddy
@Demwunz need style for messages buddy

**Comment by michaelelwin** [comment]
Copy for flash message below in blue

Claim saved. Your claim number has been emailed to you.

Return to your claim [Link]

Good to have some styling on this message so it stands out. Eg grey
background and a border.

Also feedback link is competing with it, so perhaps a case for removing it
on this page...

Michael Elwin
Digital Content Designer
07947 557671

**Comment by stevegraham** [comment]
> Claim saved. Your claim number has been emailed to you.
> Return to your claim [Link]

- "Claim saved" appears on the previous interstitial page. 
- The claim number won't be emailed if there one wasn't supplied.
- "Return to your claim" appears at the bottom of that page already

For this we just need copy that indicates the user's session has been terminated. At this point it's "you've been logged out"

**Comment by Demwunz** [comment]
styling is all done. may need a once over from Eddie

**Comment by michaelelwin** [comment]
If user enters an email address they do not see an interstitial page; they
just get dumped on the start page, which is confusing.

So this flash message is an opportunity to communicate that the claim is
saved and where they can retrieve it.

Ideally we would have an interstitial page which says 'claim saved' etc,
but that is not in the current build.

So can we have two versions of the flash message for different conditions?




Michael Elwin
Digital Content Designer
07947 557671

**Comment by stevegraham** [comment]
Please fix merge conflicts

**Comment by stevegraham** [comment]
@michaelelwin i see what you mean. yes, as long as the message is one line.

File: app/assets/stylesheets/elements/_flash-messages.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.flash {
+  background-color:$highlight-colour;
+  border-width:0 0 0 8px;
+  border-style:solid;
+  margin: $gutter 0;
+  padding:$gutter 0;
+  &.info {
+    border-color:$grass-green;
```
Comment (stevegraham) [inline app/assets/stylesheets/elements/_flash-messages.scss]: Green isn't working for me.

Comment (michaelelwin) [inline app/assets/stylesheets/elements/_flash-messages.scss]: *FLASH MESSAGE COPY *


*IF EMAIL ADDRESS SUPPLIED (coming from any page, having clicked 'save and
complete later')*

Claim saved. Your claim number has been emailed to you. *Return to your
claim* [Link]


*IF NO EMAIL ADDRESS SUPPLIED (coming from interstitial page)*

You have been logged out. *Return to your claim.* [Link]


*NOTE, NEW COPY / DESIGN FOR INTERSTITIAL PAGE (IF NO EMAIL ADDRESS
SUPPLIED) IN* GOOGLE DOC
<https://docs.google.com/a/digital.justice.gov.uk/document/d/1h0V_N7wWygMceT7bmTptDoK1WTW0DeMmWqNOIKX_i08/edit>



Michael Elwin
Digital Content Designer
07947 557671

**Comment by stevegraham** [comment]
Merge conflicts need to be fixed

**Comment by Demwunz** [comment]
@stevegraham fixed merge conflicts.

**Comment by stevegraham** [comment]
Wonderful. :love_hotel:

**Pull request merged**